### ISSUE TITLE
Java ProcessBuilder and backslashing on Windows 😿

### ISSUE THREAD

**Issue by excitoon** [bazelbuild/bazel#8231]
Example is here: https://github.com/excitoon/bazel-issues/tree/master/processbuilder-on-windows

On Windows, `ProcessBuilder` does not make a good job joining list of arguments into correct command line.
If one passes backslashes inside of an argument of `ctx.execute`, then these will be spoiled:

```
> bazel build @some_name//:some_file.txt
DEBUG: C:/users/chebotarev_v/documents/src/bazel-issues/processbuilder-on-windows/some_rule.bzl:7:3: \\two\\two\\\thr
ee\\\\four\one
DEBUG: C:/users/chebotarev_v/documents/src/bazel-issues/processbuilder-on-windows/some_rule.bzl:13:3: \\\two\\\two\\\
\\three\\\\\\\four\one
```

Real command line (from Task Manager):

```
"python3" test.py \\\two\\\two\\\\\three\\\\\\\four\one
```

Look how two backslashes convert to three, three to five, four to seven, by `2*N-1` formula.

See also: https://stackoverflow.com/questions/34402429/processbuilder-does-something-to-my-command-line-arguments-on-windows .

**Comment by laszlocsomor** [comment]
Thanks for filing and for the repro!
Looks like https://github.com/bazelbuild/bazel/issues/7486 might solve this.

Does it repro with Bazel 0.25.0, or with 0.24.0 and the `--incompatible_windows_escape_jvm_flags` *startup* flag? (i.e. it goes between `bazel` and `build`)

**Comment by laszlocsomor** [comment]
Ah never mind, this is a different bug. Let me take a closer look.

**Comment by laszlocsomor** [comment]
Sorry about the silence.

This bug is fixed in the upcoming Bazel 0.26.0 release.
You can try the release candidate: https://releases.bazel.build/0.26.0/rc8/index.html

**Comment by excitoon** [comment]
Yeah, that was #7454 .
